bcda/models/service.go

```diff
@@ -174,13 +174,13 @@ func (s *service) getBenes(cclfFileID uint) ([]*CCLFBeneficiary, error) {
 
 // IsSupportedACO determines if the particular ACO is supported by checking
 // its CMS_ID against the supported formats.
-func IsSupportedACO(cmsID string) (bool, error) {
+func IsSupportedACO(cmsID string) bool {
 	const (
 		ssp     = `^A\d{4}$`
 		ngaco   = `^V\d{3}$`
 		cec     = `^E\d{4}$`
 		pattern = `(` + ssp + `)|(` + ngaco + `)|(` + cec + `)`
 	)
 
-	return regexp.MatchString(pattern, cmsID)
+	return regexp.MustCompile(pattern).MatchString(cmsID)
```

Comment:
Since we're using a constant regex, updated this to be a fatal error instead of relying on caller to throw the correct error.

Since this method is well-tested, we should not encounter and panics from this method.